chcemvediet/apps/anonymization/migrations/0002_attachmentrecognition.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('attachments', '0004_name_sanitization'),
```

Comment:
Automaticky mi vygenerovalo migraciu kde ako dependency su az dve migracie, co sa mi este nestalo. Je to OK ?

Reply:
Ano, je bezne, ze migracia ma viac dependencies. Zvycajne dava ako dependency predoslu migraciu v poradi v danej app, plus poslednu migraciu tykajucu sa modelov na ktore mame FK. Ked si pozries migracie v ostatnych apps, tak niekde najdes aj tri dependencies.